echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/orca/PipelineInitiator.java

```diff
@@ -153,8 +154,20 @@ private void onOrcaError(Pipeline pipeline, Throwable error) {
   }
 
   private static boolean isRetryable(Throwable error) {
-    return error instanceof RetrofitError &&
-      (((RetrofitError) error).getKind() == Kind.NETWORK || ((RetrofitError) error).getKind() == Kind.HTTP);
+    if (!(error instanceof  RetrofitError)) {
+      return false;
+    }
+    RetrofitError retrofitError = (RetrofitError) error;
+
+    if (retrofitError.getKind() == Kind.NETWORK) {
+      return true;
+    }
+
+    if (retrofitError.getKind() == Kind.HTTP && retrofitError.getResponse().getStatus() != HttpStatus.BAD_REQUEST.value()) {
```

Comment:
Are we sure that `.getResponse` is not `null`?

Reply:
Good point; it shouldn't be as `Kind.HTTP` explicitly means that it received a non-200 response, but that's not really statically enforced so I fixed it to be null-safe.